spec/components/previews/user/avatar_component_preview.rb

```diff
@@ -0,0 +1,6 @@
+class User::AvatarComponentPreview < ViewComponent::Preview
```

Comment:
Wouldn't work without inlining the namespace. Odd. Perhaps related to devise or other things fighting for the module namespace, but I would have expected 

```rb
module User
 class FooPreview
 end
end
```
to work :shrug:

Reply:
Yeah I reckon it's the `User` model class conflicting with the `User` module here. I usually use a plural version for namespaces to get around it:
```ruby
module Users
  class FooPreview 
  end 
end
```